src/topology/homotopy/fundamental_group.lean

```diff
@@ -0,0 +1,58 @@
+/-
+Copyright (c) 2021 Mark Lavrentyev. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mark Lavrentyev
+-/
+import category_theory.category.Groupoid
+import category_theory.groupoid
+import topology.category.Top.basic
+import topology.path_connected
+import topology.homotopy.path
+import topology.homotopy.fundamental_groupoid
+
+/-!
+# Fundamental groupoid of a space
+
+Given a topological space `X` and a basepoint `x`, the fundamental group is the automorphism group
+of `x` i.e. the group with elements being loops based at `x` (quotiented by homotopy equivalence).
+-/
+universes u v
+
+variables {X : Type u} {Y : Type v} [topological_space X] [topological_space Y]
+variables {x₀ x₁ : X}
+
+noncomputable theory
+
+/-- The fundamental group is the automorphism group (vertex group) of the basepoint
+in the fundamental groupoid.
+-/
+def fundamental_group (X : Type u) [topological_space X] (x : X) :=
+@category_theory.Aut
+  X
+  (@category_theory.groupoid.to_category (fundamental_groupoid X) _)
+  x
```

Comment:
I think that we should have a name (e.g., `fundamental_groupoid.mk`) for the identity map from `X` to `fundamental_groupoid X`. Then this will be `category_theory.Aut (fundamental_groupoid.mk x)`.

Reply:
We have `from_top` (https://github.com/leanprover-community/mathlib/blob/eae6ae374c51f6400ab0bac5ec2ada12908fb288/src/topology/homotopy/fundamental_groupoid.lean#L341) but when I wrote that, I was visualizing it more as "just helping the typechecker" (i.e. essentially a type hint). For example, I often want to write `x \hom y` where `x` and `y` are actually points in a topological space, to mean the homotopy class of paths between `x` and `y`.  But this doesn't work unless you do something like `from_top x \hom from_top y`.

Reply:
@urkud Is this not what `fundamental_groupoid X` already does? It's basically the identity map as is. (definition below for context)
```
def fundamental_groupoid (X : Type u) := X
```